objects/official/ride/rct2.dlc.zpanda/object.json

```diff
@@ -0,0 +1,876 @@
+        { "path": "images/000688.png","x": -12, "y": -16 },
+        { "path": "images/000689.png","x": -12, "y": -17 },
+        { "path": "images/000690.png","x": -11, "y": -17 },
+        { "path": "images/000691.png","x": -11, "y": -17 },
+        { "path": "images/000692.png","x": -11, "y": -17 },
+        { "path": "images/000693.png","x": -10, "y": -17 },
+        { "path": "images/000694.png","x": -9, "y": -17 },
+        { "path": "images/000695.png","x": -9, "y": -16 },
+        { "path": "images/000696.png","x": -7, "y": -15 },
+        { "path": "images/000697.png","x": -6, "y": -14 },
+        { "path": "images/000698.png","x": 45, "y": 44 },
+        { "path": "images/000699.png","x": 45, "y": 44 },
+        { "path": "images/000700.png","x": 45, "y": 44 },
+        { "path": "images/000701.png","x": 45, "y": 44 },
+        { "path": "images/000702.png","x": 6, "y": -8 },
+        { "path": "images/000703.png","x": -6, "y": -16 },
+        { "path": "images/000704.png","x": -1, "y": -16 },
+        { "path": "images/000705.png","x": -9, "y": -11 },
+        { "path": "images/000706.png","x": -2, "y": -9 },
+        { "path": "images/000707.png","x": -6, "y": -11 },
+        { "path": "images/000708.png","x": -8, "y": -16 },
+        { "path": "images/000709.png","x": -9, "y": -16 },
+        { "path": "images/000710.png","x": 1, "y": -9 },
+        { "path": "images/000711.png","x": -2, "y": -17 },
+        { "path": "images/000712.png","x": -1, "y": -14 },
+        { "path": "images/000713.png","x": -11, "y": -11 },
+        { "path": "images/000714.png","x": 45, "y": 44 },
+        { "path": "images/000715.png","x": -2, "y": -11 },
+        { "path": "images/000716.png","x": -8, "y": -14 },
+        { "path": "images/000717.png","x": -11, "y": -17 },
+        { "path": "images/000718.png","x": 45, "y": 44 },
+        { "path": "images/000719.png","x": -2, "y": -13 },
+        { "path": "images/000720.png","x": -4, "y": -16 },
+        { "path": "images/000721.png","x": -9, "y": -16 },
+        { "path": "images/000722.png","x": -10, "y": -13 },
+        { "path": "images/000723.png","x": -2, "y": -13 },
+        { "path": "images/000724.png","x": -4, "y": -16 },
+        { "path": "images/000725.png","x": -9, "y": -16 },
+        { "path": "images/000726.png","x": -10, "y": -13 },
+        { "path": "images/000727.png","x": -2, "y": -13 },
+        { "path": "images/000728.png","x": -4, "y": -16 },
+        { "path": "images/000729.png","x": -9, "y": -16 },
+        { "path": "images/000730.png","x": -5, "y": -13 },
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        ""
```

Comment:
What do all of these mean?

Reply:
Empty images that need to be there because the game expects them.

Reply:
This seems very hacky, though. Would implementing an empty fallback on the cpp side be an option?

Reply:
As far as I know, there is a fallback, but it results in error messages like "image table longer/shorter than expected".

Reply:
OK, I have done a test. They are used for the invisible cars at the begin and end. If you remove them, you will get strange artefacts, especially when setting the colour for every car.

Reply:
Thanks for checking and explaining.

It's a shame JSON doesn't support comments (at least not in the  standard), because this seems like a good occasion for some. Perhaps you could add a brief explanation to the commit message at least.

Reply:
> It's a shame JSON doesn't support comments (at least not in the standard), because this seems like a good occasion for some.

I fully agree - especially for images. I ran into the same problem when porting RCT1 vehicles over - it would have been so much easier if I could just group them with a comment.